docs/troubleshooting.rst

```diff
@@ -48,3 +48,21 @@ or file an issue on `GitHub
 <https://github.com/kanisterio/kanister/issues>`_. A `mailing list
 <https://groups.google.com/forum/#!forum/kanisterio>`_ is also
 available if needed.
+
+Validating webhook for Blueprints
+=================================
+For the validating webhook to work, the Kubernetes API Server needs to
+connect to port ``9443`` of the Kanister operator. If your cluster has
+a firewall setup, it has to be configured to allow that communication.
+
+GKE
+---
+If you get an error, that the webhook can't be reached you need to check if
+your firewall misses a rule for port 9443:
```

Comment:
```suggestion
your firewall misses a rule for port ``9443``:
```

Reply:
It will be helpful to mention where this error is from i.e. the Kanister operator logs.

Reply:
@muffl0n I think this is not addressed.

Reply:
The error (and the command that precedes it) is in the code block below. I feel that this is explanation enough, isn't it?

Reply:
To confirm, is the error part of the kubectl output or the operator logs?

Reply:
It's the kubectl output. As the kubernetes control plane is not able to reach the webhook, it can't get any output of the operator, yet.

Reply:
I still think that 9443 should be ``9443`` but its very minor. I think we can get the PR in.

Reply:
Agreed - for consistency sake.

Reply:
Thanks for the reminder, I missed the part with the backticks. Just added them.